server/index.js

```diff
@@ -237,6 +243,31 @@ export default class Server {
     }
   }
 
+  async readBuildId () {
+    const buildIdPath = join(this.dir, '.next', 'BUILD_ID')
+    try {
+      const buildId = await fs.readFile(buildIdPath, 'utf8')
+      return buildId.trim()
+    } catch (err) {
+      if (err.code === 'ENOENT') {
+        return '-'
```

Comment:
Is it difficult to create `BUILD_ID` on dev mode too? I think it should work the same on dev and production as much as possible otherwise you might find problems on production.